keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/SecurityGroupHandler.kt

```diff
@@ -119,46 +119,20 @@ open class SecurityGroupHandler(
         .map { it.await() }
     }
 
-  override suspend fun export(exportable: Exportable): SecurityGroupSpec {
-    val summaries = exportable.regions.associateWith { region ->
-      try {
-        cloudDriverCache.securityGroupByName(
-          account = exportable.account,
-          region = region,
-          name = exportable.moniker.toString()
-        )
-      } catch (e: ResourceNotFound) {
-        null
+  override suspend fun actuationInProgress(resource: Resource<SecurityGroupSpec>): Boolean =
```

Comment:
No change here, I just moved it so that the export-related methods sit together in the file.